src/site/content/en/blog/speed-at-scale/index.md

```diff
@@ -0,0 +1,85 @@
+
+## Lighthouse now supports Performance Budgeting
+
+[LightWallet](https://developers.google.com/web/tools/lighthouse/audits/budgets) is a new feature in Lighthouse that adds support for [performance budgets](/fast#enforce-performance-budgets). Performance budgets establish standards for the performance of your site. More importantly, they make it is easy to identify and fix performance regressions before they ship.
+
+<figure class="w-figure">
+  <img src="./Speed-at0.png" alt="LightWallet performance budgets" loading="lazy">
+</figure>
+
+LightWallet is available in the newest version of the Lighthouse CLI and only takes a couple minutes to set up. These[ instructions](https://developers.google.com/web/tools/lighthouse/audits/budgets) provide more information.
+
+Unsure what your budgets should be? Try our experimental [Performance Budget Calculator](https://bit.ly/perf-budget-calculator) which can generate a LightWallet compatible budget configuration.
+
+## Native image and iframe lazy-loading comes to the web
+
+Web pages often contain a large number of images, which contribute to data-usage, [page-bloat](https://httparchive.org/reports/state-of-images) and slower page loads. Many of these images are offscreen, requiring a user to scroll in order to view them. 
+
+Until now, you've needed to solve lazy-loading images using a JavaScript library but that may soon change. This summer, Chrome will be launching support for the [loading](https://addyosmani.com/blog/lazy-loading/) attribute which brings native `<img>` and `<iframe>` lazy-loading to the web.
+
+<figure class="w-figure">
+  <img src="./Speed-at1.png" alt="Native lazy-loading highlighting offscreen content being loaded on-demand" loading="lazy">
+</figure>
+
+The `loading` attribute allows a browser to defer loading offscreen images and iframes until users scroll near them. `loading` supports three values:
+
+* `lazy`: is a good candidate for lazy loading.
+* `eager`: is not a good candidate for lazy loading. Load right away.
+* `auto`: browser will determine whether or not to lazily load.
+
+```html
+<img src="io2019.jpg" loading="lazy" alt="..." />
+<iframe src="video-player.html" loading="lazy"></iframe>
+```
+
+The exact heuristics for "when the user scrolls near" is left up to the browser. In general, our hope is that browsers will start fetching deferred images and iframe content a little before it comes into the viewport. 
+
+The `loading` attribute is implemented behind flags in Chrome Canary. You can try out [this demo](https://mathiasbynens.be/demo/img-loading-lazy) in Chrome 75+ with the `chrome://flags/#enable-lazy-image-loading` and `chrome://flags/#enable-lazy-frame-loading` flags turned on. 
+
+A [write-up](https://addyosmani.com/blog/lazy-loading/) on the native lazy-loading feature is available with more details.
+
+## Google fonts now supports font-display as a query parameter
```

Comment:
```suggestion
## Google Fonts now supports font-display as a query parameter
```

It looks like they capitalize it on their site and having it be a proper noun makes the word "supports" read a bit better IMO